components/Recommendations/Recommendations.tsx

```diff
@@ -298,6 +298,7 @@ const useStyles = () => {
         default: {
           borderBottomWidth: 0.5,
           borderBottomColor: itemSeparatorColor(colorScheme),
+          paddingLeft: 16,
```

Comment:
a BIG refactor but would be amazing to start using the theme 😄 
`export const Paddings = {
  default: 16,
  small: 8,
  large: 24,
};`